tests/devices/test_lightning_qubit.py

```diff
@@ -112,14 +112,8 @@ class TestDtypePreserved:
                 qml.expval(qml.PauliY(0)), marks=pytest.mark.xfail(reason="incorrect type")
             ),
             pytest.param(qml.var(qml.PauliY(0)), marks=pytest.mark.xfail(reason="incorrect type")),
```

Comment:
These measurements are returning vanilla python floats I believe. We discussed potentially allowing that behaviour. Is that something we strictly want to avoid with `lightning.qubit`? If yes, we should probably also add changes to enforce that and update these test cases accordingly.

Reply:
I went to quickly on those tests. The circuit is returning a Python float, and hence the assert fails when trying to access the `dtype` attribute. I'll fix that.